### ISSUE TITLE
Trained weights

### ISSUE THREAD

**Issue by ofirkris** [lj107024/FDDWNet#2]
Thank you for your hard work, can you share trained weights?

**Comment by lj107024** [comment]
The weight setting refers to other code, and weights are calculated through all the dataset images, which can be seen in the loss.py.
`weight = torch.FloatTensor(
                 [2.8149, 6.9850, 3.7890, 9.9428, 9.7702, 9.5110, 10.3113, 10.0264,
                 4.6323, 9.5608, 7.8698, 9.5168, 10.3737, 6.6616, 10.2604, 10.2878,
                 10.2898, 10.4053, 10.13809])`

**Comment by ofirkris** [comment]
Thanks, can you share the pre-trained model?
